<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>439</width>
    <height>347</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Imaginote</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>421</width>
      <height>323</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="loadSpaceButton">
         <property name="text">
          <string>Load space...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="spaceLabel">
         <property name="text">
          <string>SpaceDef</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Observer's head position angle</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="headPosLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="presIntervalLabel">
         <property name="text">
          <string>Object presentation interval [s]</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="presIntervalLineEdit">
         <property name="text">
          <string>PresInterval</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Wave type</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="waveTypeCombo">
         <item>
          <property name="text">
           <string>Sinusoidal</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Triangular</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Sound pitch</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="soundPitchCombo">
         <item>
          <property name="text">
           <string>Do not use</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Distance</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Sound timbre (2th / 3th ratio)</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QComboBox" name="soundTimbreCombo">
         <item>
          <property name="text">
           <string>Do not use</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Width / height ratio</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="startStopButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="listenerPosLabel">
       <property name="text">
        <string>Listener position: undefined</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="listenerAngleLabel">
       <property name="text">
        <string>Listener angle: undefined</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>loadSpaceButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>loadSpaceDef()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>215</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>37</x>
     <y>23</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>startStopButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>startStopClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>307</x>
     <y>220</y>
    </hint>
    <hint type="destinationlabel">
     <x>307</x>
     <y>263</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>presIntervalLineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>validatePresInterval(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>308</x>
     <y>167</y>
    </hint>
    <hint type="destinationlabel">
     <x>50</x>
     <y>183</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>loadSpaceDef()</slot>
  <slot>startStopClicked()</slot>
  <slot>validatePresInterval(QString)</slot>
 </slots>
</ui>
